app.rb

```diff
@@ -171,75 +171,4 @@ def list_person_rentals
       end
     end
   end
-
-  class App
-    include CreateEntity
-    include DisplayEntity
-
-    def initialize
-      @all_books = []
-      @all_people = []
-      @all_rentals = []
-    end
-
-    def run
-      loop do
-        print_menu
-        option = gets.chomp.to_i
-        puts "\n"
-        process_option(option)
-        break if option == 7
-      end
-    end
-
-    def process_option(option)
-      case option
-      when 1
-        list_all_books
-      when 2
-        list_all_people
-      when 3
-        create_person
-      when 4
-        create_book
-      when 5
-        create_rental
-      when 6
-        list_person_rentals
-      else
-        puts "Invalid option! Please choose a valid option.\n\n"
```

Comment:
- [ ] Optional: I would highly suggest to add a case for option 7, before quitting it should print 'Thank you for using the app' etc. rather than 'invalid option'. This is for matching the design and making the app more professional.